docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,355 @@
+# Tutorial: Get started with oras-go v2
+
+This tutorial introduces the basics of managing OCI artifacts with the [oras-go v2](https://pkg.go.dev/oras.land/oras-go/v2) package.
+
+You'll get the most out of this tutorial if you have a basic familiarity with Go and its tooling. If this is your first exposure to Go, please see [Tutorial: Get started with Go](https://golang.org/doc/tutorial/getting-started) for a quick introduction.
+
+The tutorial includes the following sections:
+
+1. Create a folder for your code.
+2. Connect to a remote repository.
+3. Show tags in the repository.
+4. Push a layer to the repository.
+5. Push a manifest to the repository.
+6. Fetch the manifest from the repository.
+7. Parse the fetched manifest content and get the layers.
+8. Copy the artifact from the repository.
+
+The complete code is provided at the end of this tutorial.
+
+## Create a folder for your code
+
+Open a command prompt and cd to your home directory.
+
+On Linux or Mac:
+```shell
+cd
+```
+On Windows:
+```shell
+cd %HOMEPATH%
+```
+
+Create a directory for your code.
+```shell
+mkdir oras-go-v2-quickstart
+cd oras-go-v2-quickstart
+```
+Create a module in which you can manage dependencies.
+
+```shell
+go mod init quickstart/oras-go-v2
+go: creating new go.mod: quickstart/oras-go-v2
+```
+
+In your text editor, create a file `main.go` in which to write your code.
+
+## Connect to a remote repository with basic authentication
+
+Paste the following into `main.go` and save the file. This code demonstrates how to use [NewRepository](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote#NewRepository) from the [remote](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote) package to connect to a remote repository. Basic authentication (using a username and password) is handled by the [auth](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote/auth) package. Other authentication methods, such as refresh token authentication, are also supported.
```

Comment:
use the relative package path like `registry/remote` and `registry/remote/auth` instead of `remote` and `auth`.

Reply:
same suggestion for the rest of the doc.